lib/ui/views/security_view.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+import 'package:notredame/core/viewmodels/security_viewmodel.dart';
+import 'package:notredame/generated/l10n.dart';
+import 'package:notredame/ui/utils/app_theme.dart';
+import 'package:stacked/stacked.dart';
+
+import 'emergency_view.dart';
+
+class SecurityView extends StatefulWidget {
+  @override
+  _SecurityViewState createState() => _SecurityViewState();
+}
+
+class _SecurityViewState extends State<SecurityView> {
+  static const CameraPosition _etsLocation = CameraPosition(
+      target: LatLng(45.49449875, -73.56246144109338), zoom: 17.0);
+
+  @override
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<SecurityViewModel>.reactive(
+        viewModelBuilder: () => SecurityViewModel(),
+        builder: (context, model, child) => Scaffold(
+          appBar: AppBar(
+            title: Text(AppIntl.of(context).ets_security_title),
+          ),
+          body: SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Container(
+                  height: 250,
+                  child: GoogleMap(
+                      initialCameraPosition: _etsLocation,
+                      zoomControlsEnabled: false,
+                      markers: model.getMarkers(model.markersList),
+                      gestureRecognizers: <
+                          Factory<OneSequenceGestureRecognizer>>{
+                        Factory<OneSequenceGestureRecognizer>(
+                            () => EagerGestureRecognizer()),
+                      }),
+                ),
+                Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: Text(
+                    AppIntl.of(context).security_reach_security,
+                    style: const TextStyle(
+                        color: AppTheme.etsLightRed, fontSize: 24),
+                  ),
+                ),
+                Card(
+                  child: InkWell(
+                    splashColor: Colors.red.withAlpha(50),
+                    onTap: () => model.makePhoneCall(
+                        'tel:${AppIntl.of(context).security_emergency_number}'),
```

Comment:
You should handle the possible error. In the case, the phone application is not available an exception is thrown and so the application will crash, we should display a Toast or a snackbar to handle the error 😄  (you can look [here](https://github.com/ApplETS/Notre-Dame/blob/master/lib/ui/views/login_view.dart#L110) to have an example)

Reply:
done